src/functions-and-arrays.js

```diff
@@ -1,20 +1,50 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(numb1,numb2) {
+
+  if (numb1 > numb2){
+return numb1;
+}
+else if(numb1 < numb2){
+  return numb2;
+} else if (numb1 === numb2){
+  return numb1 || numb2;
+}
+}
 
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(arr) {
+  if(arr.length === 0){
+    return null;
+  }else if (arr.length === 1){
+      return arr[0];
+    }
+Math.max(...(x.map(el => el.length)));
+
+// for (let i = 0; i<)
+}
 
 
 
 // Iteration #3: Calculate the sum
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
-function sumNumbers() {}
+const initVal = 0;
+
+function sumNumbers(arr) {
+const result = numbers.reduce((previousValue, currentValue) => previousValue + currentValue, initVal);
+if(arr.length === 0){
+  return 0;
```

Comment:
You will learn this array method tomorrow. Cool that you tried it, but this task would have been possible even with the stuff you have already learned